File: ejercicio_4.java
import java.util.Scanner;

public class ejercicio_4 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        System.out.print("Ingrese su peso en kilogramos: ");
        double peso = entrada.nextDouble();
        System.out.print("Ingrese su altura en metros: ");
        double altura = entrada.nextDouble();

        double imc = peso / (altura * altura);

        System.out.println("Su IMC es: " + imc);

        if (imc <= 18.5) {
            System.out.println("Bajo peso");
        } else if (imc <= 25.0) {
            System.out.println("Normal");
        } else if (imc <= 30.0) {
            System.out.println("Sobrepeso");
        } else {
            System.out.println("Obeso");
        }

    }
}

File: ejercicio_4.py
peso = float(input("Ingresa tu peso en kilogramos: "))
altura = float(input("Ingresa tu altura en metros: "))

imc = float(peso / (altura * altura))

print("Su IMC es: " ,imc)

if  imc <= float(18.5):
    print("Bajo peso")
elif imc <= float(25.0):
    print("Normal")
elif imc <= float(30.0):
    print("sobre peso")
elif imc > float(30.0):
    print("Obeso")

